infra/aws/terraform/prow-build-cluster/locals.tf

```diff
@@ -39,6 +39,7 @@ locals {
   auto_scaling_tags = {
     "k8s.io/cluster-autoscaler/${var.cluster_name}" = "owned"
     "k8s.io/cluster-autoscaler/enabled"             = true
+    "karpenter.sh/discovery"                        = var.cluster_name
```

Comment:
I don't think we want this here - we want this on the security group that Karpenter is going to use (i.e. - `node_security_group_tags`) and the subnets it will use for discovery. In terms of the security group though, this tag should only be on one security group

Reply:
Thanks, fixed!